packages/server/lib/plugins/preprocessor.coffee

```diff
@@ -36,11 +37,25 @@ baseEmitter = new EE()
 fileObjects = {}
 fileProcessors = {}
 
-setDefaultPreprocessor = ->
+setDefaultPreprocessor = (config) ->
   debug("set default preprocessor")
 
   browserify = require("@cypress/browserify-preprocessor")
-  plugins.register("file:preprocessor", browserify())
+
+  options = browserify.defaultOptions;
```

Comment:
So far, our strategy has been to keep any default options in [`@cypress/browserify-preprocessor`](https://github.com/cypress-io/cypress-browserify-preprocessor) instead of changing those options here. I think we should keep with that strategy here. So this code and the files it relies on would move to a new PR against the browserify preprocessor and then this PR just needs to bump the version in `server/package.json`.

Reply:
I moved code. Please check cypress-io/cypress-browserify-preprocessor#38.